src/app/new-page/component/content-accordion/content-accordion.component.css

```diff
@@ -7,59 +15,57 @@
   width: 100%;
   text-align: left;
   font-size: 16px;
+  font-family:
+    Source Sans Pro,
+    sans-serif !important;
   border: none;
   outline: none;
   transition: 0.4s;
-  color: var(--kg-white);
-  background-color: var(--kg-blue);
-  transition: all 0.3s ease 0s;
+  color: var(--kg-gray-dark);
+  background-color: var(--kg-gray-light);
+  transition: all 0.3s linear;
   position: relative;
+  display: flex;
+  align-items: center;
 }
 
 .accordion-header:hover {
-  background-color: var(--kg-blue-light);
+  background-color: var(--kg-gray);
+  color: var(--kg-gray-dark);
+}
+.accordion-header:active {
+  background-color: var(--kg-gray-medium-dark);
```

Comment:
On mobile, we might need to make the padding less when get below 400px. This isn;t urgent as they could use the App when on smaller devices.